README.md

```diff
@@ -64,6 +64,15 @@ Every
 
 Should have a corresponding documentation.
 
+## Shape all sound in one sound with sprites using audiosprite
+
+Command line in the sound folder : `audiosprite $(find . -type f -name '_.mp3' -o -name '_.wav') -o mybruteaudio -u .`
```

Comment:
```suggestion
- Run the following command in the sound folder : `audiosprite $(find . -type f -name '_.mp3' -o -name '_.wav') -o mybruteaudio -u .`
```

Please rename mybryteaudio to sfx here too (and the file created too).
Also, it would be better to add a script in package.json, and call it at the root (`yarn sfx:generate`) for example, which would execute the above command in the correct folder.